lib/ReactViews/Map/Panels/SharePanel/BuildShareLink.ts

```diff
@@ -110,7 +110,7 @@ export async function buildShortShareLink(
 export function getShareData(
   terria: Terria,
   viewState?: ViewState,
-  options = { includeStories: true }
+  options = { includeStories: viewState?.includeStoryInShare ?? true }
```

Comment:
Each of the share link/share data functions should have the same defaults. And after I think you should revert all changes made to this file.

Reply:
Implemented